package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cuttle",
-  "version": "4.3.20",
+  "version": "4.3.22",
```

Comment:
Going forward, let's refrain from updating package-lock.json's version field manually. It updates to match `package.json` whenever we run `npm install`, which means that we can use the version here as a record of the last time that command was run. This is fine for now, but we can leave it off next time